editions/2015-06.md

```diff
@@ -0,0 +1,33 @@
+Ahoy there and welcome to the fifth edition of Civic Tech Monthly. Below you’ll find news and notes about civic tech from Australia and around the world.
+
+We have a slightly smaller bounty of news items this month. So it's a great time to remind you that the entire production of Civic Tech Monthly is done [in the open on GitHub](https://github.com/openaustralia/newsletter). We very much welcome suggestions throughout the month for items to add to the newsletter.
```

Comment:
Actually this issue has a piracy item in it so it could easily be misconstrued. I'll de-sea these references.